### PR TITLE
Fixes #747 text to speech works now

### LINKED ISSUE #747: Speech to text not working
speech to text is not working due to pause in the speech.

### PR DESCRIPTION
**Pull request by nikhilrayaprolu**
Fixes issue #747 ,
now text to speech works.

Demo Link: http://susper-pr-748.herokuapp.com/

### CODE REVIEW

**Comment by harshit98** [comment]
Please edit the PR message: "Text-To-Speech"

**Comment by harshit98** [comment]
@nikhilrayaprolu please provide a video for it. On heroku link, it seems to be not working.

**Comment by Marauderer97** [comment]
@nikhilrayaprolu Yes like @harshit98 suggested, please provide a gif/video link

**Comment by nikhilrayaprolu** [comment]
@harshit98 @Marauderer97 please check now, there is a slight modification of code

File: src/app/infobox/infobox.component.ts [hunk 1]
```diff
@@ -56,8 +56,10 @@ export class InfoboxComponent implements OnInit {
   }
 
   startSpeaking(description) {
-    this.synthesis.speak(description);
-    this.synthesis.pause();
+    let msg = new SpeechSynthesisUtterance(description);
+    window.speechSynthesis.cancel();
+    window.speechSynthesis.cancel();
```
Comment (harshit98) [inline src/app/infobox/infobox.component.ts]: why `cancel()` is being called two times?

**Comment by Marauderer97** [comment]
@nikhilrayaprolu Can we have a video/gif please?

**Comment by mariobehling** [comment]
What about text to speech? I don't get a sound for the answer.

**Comment by harshit98** [comment]
@nikhilrayaprolu please solve Travis issue.

**Comment by mariobehling** [comment]
Not working for me.

**Comment by nikhilrayaprolu** [comment]
@mariobehling updated the pull and tested it on my friend's laptops too

**Comment by Marauderer97** [comment]
@nikhilrayaprolu Not working for me either, I am not sure I am using the correct procedure, a gif/video will help :+1:

**Comment by nikhilrayaprolu** [comment]
@Marauderer97 I am not able to record both system audio + screen on my laptop, The applications @harshit98 shared are giving me some errors. please look at these videos which are recorded in some hacky way.
https://www.youtube.com/watch?edit=vd&v=N8wDKUGZI10
https://www.youtube.com/watch?v=kFA-bbx4KKI&feature=youtu.be

**Comment by mariobehling** [comment]
Thanks. I need to test this more in detail on different browsers. Merging this. Cheers.

**Pull request merged**